Makefile

```diff
@@ -36,6 +36,7 @@ CLASSES += src/malva/java/net/NetworkInterfaceTest.class
 CLASSES += src/malva/java/nio/DirectByteBufferTest.class
 CLASSES += src/malva/java/util/HashMapTest.class
 CLASSES += src/malva/java/util/regex/PatternTest.class
+CLASSES += src/malva/java/text/DecimalFormatTest.class
```

Comment:
Maybe try to add new entries in alphabetical order (although the existing list does not completely follow this)